apps/css-workshop/src/pages/tag.astro

```diff
@@ -22,31 +22,24 @@ import Button_ from '../components/Button.astro';
   <section id='demo-default'>
     <span class='iui-tag' id='test-tag-1'>
       <span class='iui-tag-label' title='Tag'>Tag</span>
-      <Button_
-        class='iui-tag-button'
-        variant='borderless'
-        size='small'
-        aria-label='Delete tag'
-        type='button'
-      >
-        <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-      </Button_>
+      <IconButton_ class='iui-tag-button' variant='borderless' size='small' aria-label='Delete tag'>
```

Comment:
This file and the others that use the Tag component should use that instead of this.

Reply:
Can you not just keep your original changes in the case of a merge conflict and then edit the Tag component with the updated Button component? I can just hold the merge until the Tag component is done, but I don't think the conflict here would be too difficult to solve without having to redo work.

Reply:
All I know is that last time I had to redo everything when I updated my branch to be in sync with main. You're probably right though

Reply:
#1966 is merged now, make sure to resolve merge conflicts

Reply:
Okay, will do.

Reply:
Merge conflicts resolved, and added `IconButton` to `Tag` in place of the `Button` component.